script.service.ltvpl/resources/data/debugFileLocation.py

```diff
@@ -0,0 +1 @@
+DEBUGCACHEFILE = r"D:\My Documents\GitHub\LiveTV_PlayList\script.service.ltvpl\resources\data\debugcache.json"
```

Comment:
same

Reply:
This absolute path is replaced at runtime by a relative path when the code runs under the Kodi environment.